actions/kpops-runner/action.yaml

```diff
@@ -25,16 +25,30 @@ inputs:
     required: false
   kpops-version:
     description: "KPOps version"
-    required: true
+    required: false
+    default: "latest"
+  helm-version:
+    description: "Helm version"
+    required: false
+    default: "latest"
+  token:
```

Comment:
Is there a README for the action (I guess not)? We should create one. This PR or issue?

Reply:
There is, I was just going to ask for an update to it

Reply:
oh, there is, I just forgot to update it

Reply:
I am actually surprised 😂